packages/main/src/pages/sign-up/SignUp.jsx

```diff
@@ -88,8 +88,20 @@ export default function Index() {
         }
       });
   };
+
+  const textAlign = () => {
+    if (
+      window.localStorage.myLanguage === "ar" ||
+      window.localStorage.myLanguage === "iw"
+    ) {
+      return {
+        textAlign: "right"
+      };
+    }
+  };
```

Comment:
I already saw this function being implemented by another dev
why are there multiple instances of it?

Reply:
I have deleted the repetitive function and made all necessary corrections.